lib/v2/constants/system_accounts.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+
+// TODO(n13): get correct icons from figma
+class SystemAccounts {
```

Comment:
Hey, what are these SystemAccounts used for? 
Can you leave a comment on this class explaining this.

Reply:
They're telos accounts used by Seeds system contracts

Reply:
Hmm they're pretty self explanatory - I added a comment but it's basically just repeating what's stated in the code anyway.

I am not a fan of comments in general - if they're needed, the code is unclear. And they get outdated and / or are irrelevant to begin with. 

Good code > comments.

Reply:
Yes, lazy developers update code and dont update comments. Happens all the time :)

Reply:
I just dont know what system accounts are used for? Why do we need them?